Cygwin has problems with constant SIGALRM from the smart scheduler, so disable it for cygwin

---
 xserver/os/utils.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xorg-server-1.5.1/xserver/os/utils.c
===================================================================
--- xorg-server-1.5.1.orig/xserver/os/utils.c
+++ xorg-server-1.5.1/xserver/os/utils.c
@@ -1504,7 +1504,7 @@ XNFstrdup(const char *s)
 
 #ifdef SMART_SCHEDULE
 
-#ifdef SIGVTALRM
+#if defined(SIGVTALRM) && !defined(__CYGWIN__)
 #define SMART_SCHEDULE_POSSIBLE
 #endif
 
